config/config.go

```diff
@@ -293,6 +299,18 @@ func (cc *CommonConfigurer) Configure(c *Config, rel string, f *rule.File) {
 				KindLoad: vals[2],
 			}
 
+		case "alias_kind":
+			vals := strings.Fields(d.Value)
```

Comment:
In my mind, I don't see a big gain for adding the load file path. Someone would have to update a rule to be the alias kind, then re-run gazelle just to have it update the loads. 

I do agree that it would be rare for same-named aliases to exist in multiple different files. 

Either way, supporting that functionality seems like something that could eventually be done in a follow-up if someone really wants it. I could see this functionality working with alias_kind taking two or three arguments.

Reply:
Seems reasonable, agree that it could be extended if needed.